metricbeat/beater/event.go

```diff
@@ -38,7 +34,7 @@ func (b eventBuilder) build() (common.MapStr, error) {
 
 	// Get and remove meta fields from the event created by the MetricSet.
 	indexName := getIndex(event, "")
-	typeName := getType(event, defaultType)
+	typeName := getType(event, b.moduleName+"."+b.metricSetName)
```

Comment:
@ruflin I would say for now to keep `type` and just change the value in Metricbeat, so I can use it in the Kibana dashboards. I think it's nice to have the same name across all Beats, so you can have the same story behind searching and only use different values depending on the Beat. We can also change the name of the field later.